manifest.yml.tmpl

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
this is a template file, but is there a script which consumes this and creates manifests from it?

Reply:
Yes.